### PR TITLE
Run pylint on test directory

### PR DESCRIPTION
**Pull request by bveeramani**

### CODE REVIEW

File: tests/core/array/test_basic.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 
 # pylint: disable=wrong-import-order
-import common
+import common  # pylint: disable=import-error
```
Comment (elibol) [inline tests/core/array/test_basic.py:18]: Does this directive only apply to this line?

Comment (elibol) [reply tests/core/array/test_basic.py:18]: @bveeramani bumping this.

Comment (bveeramani) [reply tests/core/array/test_basic.py:18]: Yes, it should only apply to that line

Comment (bveeramani) [reply tests/core/array/test_basic.py:18]: I'm like 95% sure but I can double-check.

**Pull request merged**